stdlib/io/src/main/ballerina/src/io/readable_character_channel.bal

```diff
@@ -64,6 +64,26 @@ public type ReadableCharacterChannel object {
         return readXmlExtern(self);
     }
 
+# Reads a property from a .properties file.
+# ```ballerina
+# string|io:Error result = readableCharChannel.readProperty(key);
+# ```
+# + key - The property key needs to read.
+# + return - The read property value or else an `io:Error`
```

Comment:
We can move doc comments to align with the function now. The issue we had earlier is now fixed

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/24978/commits/53413b53dd41b00e3fe7b02cc810f1c70def5e2c